golang/examples/main.go

```diff
@@ -15,6 +16,19 @@ func route(command string, arguments [][]byte) string {
 		result = ParseVin(arguments[0])
 	case "parseVout":
 		result = ParseVout(arguments[0])
+	case "parseHeader":
+		result = ParseHeader(arguments[0])
+	case "validateHeaderChain":
+		result = ValidateHeaderChain(arguments[0])
+	case "prove":
+		// convert argument to a uint
+		str := string(arguments[6])
+		uint64Arg, err := strconv.ParseUint(str, 10, 32)
+		if err != nil {
+			result = "Error converting arg to uint"
```

Comment:
need to `break` out of the switch case. Otherwise this error message will never be seen because we'll go straight to Prove afterwards